packages/sitecore-jss-angular/src/services/jss-state.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, TransferState } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { JssState } from './models';
+
+@Injectable()
+export class JssStateService {
+  get state(): Observable<JssState> {
+    return this._state.asObservable();
+  }
+  private _state: BehaviorSubject<JssState>;
+
+  constructor(protected transferState: TransferState) {
+    this._state = new BehaviorSubject<JssState>(new JssState());
+  }
+
+  getStateValue() {
```

Comment:
We can use getter as well to simplify the usage
```suggestion
  get stateValue() ...
```